EcalAlCaRecoProducers/python/alcaSkimming.py

```diff
@@ -148,7 +154,7 @@
 process.load('Configuration.StandardSequences.GeometryDB_cff')
 process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
 process.load('Configuration.EventContent.EventContent_cff')
-process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
well part of the modification to get auto:run2_mc working, slipped in but can be reverted back to v2